web/client/reducers/rulesmanager.js

```diff
@@ -105,8 +105,16 @@ function rulesmanager(state = defaultState, action) {
     case LOADING:
         return assign({}, state, {loading: action.loading});
     case SET_FILTER: {
-        const {key, value} = action;
-        if (value) {
+        const {key, value, isResetField} = action;
+        if (isResetField) {
+            if (key === "rolename") {
+                return assign({}, state, {filters: {...state.filters, [key]: value, ['groupAny']: true}});
+            } else if (key === "username") {
+                return assign({}, state, {filters: {...state.filters, [key]: value, ['userAny']: true}});
+            }
+            return assign({}, state, {filters: {...state.filters, [key]: value, [key + 'Any']: true}});
```

Comment:
i think here you have to avoid to set it up to true. please test this
```suggestion
            return assign({}, state, {filters: {...state.filters, [key]: value, [key + 'Any']: undefined}});
```